package-lock.json

```diff
@@ -0,0 +1,6788 @@
+{
```

Comment:
Please cold you remove this? We use yarn so package-lock isn't required. Could you also make sure yarn.lock is up to date with the new dep?

Reply:
Done now, thanks. Am pretty new to JS, don't have a full grasp on the whole ecosystem yet.